src/engagement_db_to_analysis/engagement_db_to_analysis.py

```diff
@@ -159,6 +160,7 @@ def generate_analysis_files(user, pipeline_config, engagement_db, engagement_db_
 
     messages_traced_data = filter_messages(user, messages_traced_data, pipeline_config)
 
-    participants_traced_data_map = _fold_messages_by_uid(user, messages_traced_data)
+    messages_traced_data = run_code_computation_functions(user, messages_traced_data,
+                                                        pipeline_config.analysis_configs.dataset_configurations)
```

Comment:
```suggestion
    messages_traced_data = run_code_computation_functions(
        user, messages_traced_data, pipeline_config.analysis_configs.dataset_configurations
    )
```
In general I think this is the better way to indent, because otherwise you end up with lots of very short lines at the far right of the screen

Reply:
updated.